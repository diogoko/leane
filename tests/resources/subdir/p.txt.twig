These are {{ first_name }}'s pets:

{% for p in pets %}
  * {{ p.name }} - {{ p.kind }}
{% endfor %}