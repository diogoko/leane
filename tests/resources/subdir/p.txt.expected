These are Joey's pets:

  * Yasmine - chick
  * Duck - duck
